assets/javascripts/kitten/components/grid/lego-grid/index.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react'
+import classNames from 'classnames'
+import Masonry from 'react-masonry-component'
+import { ScreenConfig } from '../../../constants/screen-config'
+import { NUM_COLUMNS } from '../../../constants/grid-config'
+
+export const LegoGrid = ({ className, masonryProps, children, ...others }) => {
+  const [isVisible, setVisibility] = useState(false)
+
+  useEffect(() => {
+    setVisibility(true)
```

Comment:
Est-ce qu'il ne faudrait pas gérer le cas inverse au unmount ?
```suggestion
    setVisibility(true)
    
    return () => {
      setVisibility(false)
    }
```